lib/spark_api/models/floplan.rb

```diff
@@ -0,0 +1,25 @@
+module SparkApi
+  module Models
+    class FloPlan < Base
+      extend Subresource
+      self.element_name = 'floplans'
+
+      def initialize(attributes={})
+        unless attributes['Images'].nil?
+          images = []
+          thumbnails = []
+          attributes['Images'].each do |img|
+            if img["Type"].include?('thumbnail')
```

Comment:
This doesn't seem to match the [docs](http://alpha.sparkplatform.com/docs/api_services/listings/floplans#response) for this service. I'd just double check that's working as expected.

Reply:
Given the tests line up with the docs, perhaps this should be `if img["Type"] =~ /thumbnail/`?

Reply:
I'm confused about the docs, but it's probably most likely that I haven't read the Spark docs in a while 🙃 

`Images` is listed as an Expansion data type, but **Expansions** says None. I'm curious mainly because this code is checking if `Images` is present at all. It'd be nice to drop all that guard code if Images is always an array.

Reply:
This isn't Images from the Listing object, this is the Images in the FloPlans extension

Reply:
I realize that. I'm viewing the page Wade linked (apologies for the giant arrows. I don't know how to shrink them in `Preview.app`)

![alpha sparkplatform com_docs_api_services_listings_floplans](https://user-images.githubusercontent.com/116289/78578445-28988b00-77f5-11ea-9a47-ca3cf0e79af6.png)

Reply:
I think that would be a @joshcom  question. I'm getting the Images back w/o an expansion

Reply:
Good findings here. This seems to be something we overlooked in the documentation. I can confirm that the [Images attribute for flowplan is always returned](https://github.fbsdata.com/programming/api/blob/master/app/presenters/floplan_presenter.rb#L6-L19).

Reply:
That should be flagged as an array of JSON objects, and I'll get that fixed.